src/components/About/About.css

```diff
@@ -0,0 +1,11 @@
+.staff-bios-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(275px, 1fr));
```

Comment:
having played around with the app, i love your use of grid here, it looks really clean

Reply:
@Jesuitman Thanks! I've found that for pages like this, grid is much easier for me to manage than flex-box.